demo/README.md

```diff
@@ -1,5 +1,7 @@
 # Creating the Demo Database Dump
 
+> :warning: **BEFORE YOU START!** The demo dump is created from a cleaned/stripped version of the production database. Only reports under the PORI project are kept. If any changes have been made to these reports since the last dump then they must be manually reviewed by the developer creating the dump beforehand to double check nothing has been uploaded, edited or added that should not be included in public data (ex. identifiable or proprietary information).
```

Comment:
"The demo dump is created from a cleaned/stripped version of the production database. Only reports under the PORI project are kept." suggests that it's the whole production db or that the data is derived from the production db in some way - could we change the wording here to avoid suggesting that the demo db is created by some kind of processing of the prod db?

Reply:
I'm not sure I follow what you mean. The demo db is created/derived from the production db currently